src/mon/MonCommands.h

```diff
@@ -444,6 +444,12 @@ COMMAND("fs rename "
 	"name=new_fs_name,type=CephString,goodchars=" FS_NAME_GOODCHARS
 	" name=yes_i_really_mean_it,type=CephBool,req=false",
 	"rename a ceph file system", "mds", "rw")
+COMMAND("fs swap "
+	"name=fs1_name,type=CephString "
+	"name=fs2_name,type=CephString "
```

Comment:
Should we ask the `fs2_name` a `goodchars=" FS_NAME_GOODCHARS` ? IMO this should be a old fs name, while if that possible users just input a wrong one by mistake ?